core/src/main/java/de/jplag/reporting/reportobject/ReportObjectFactory.java

```diff
@@ -174,7 +175,7 @@ private void writeOverview(JPlagResult result, String path) {
                 getDate(),// dateOfExecution
                 result.getDuration(), // executionTime
                 getMetrics(result),// metrics
-                List.of()); // clusters (deactivated for now)
+                clusteringResultMapper.map(result)); // clusters (activated for now)
```

Comment:
Please remove the `(activated for now)` as this information is no longer required